frontend/public/components/masthead-toolbar.jsx

```diff
@@ -286,6 +292,11 @@ class MastheadToolbarContents_ extends React.Component {
           externalLink: true,
           href: _.get(item, 'spec.href'),
           image: <img src={_.get(item, 'spec.applicationMenu.imageURL')} alt="" />,
+          callback: () => {
+            fireTelemetryEvent('Launcher Menu Accessed', {
+              item: _.get(item, 'spec.text'),
```

Comment:
Name here is a unique identifier from the ConsoleLink CR. Let's pass it as the ID and put the text as the name.
```suggestion
              id: item.name,
              name: _.get(item, 'spec.text'),
```